src/stim/diagram/diagram_util.h

```diff
@@ -23,6 +23,14 @@
 
 namespace stim_draw_internal {
 
+
+constexpr char * X_RED = "#FF4040";
```

Comment:
I think this needs to be `constexpr const char *` to remove the compilation warnings. If they persist, switch to a `#define`.